src/ts/UIManager.ts

```diff
@@ -249,6 +252,10 @@ export class UIManager {
       this.uiContainerElement = new DOM(player.getContainer());
     }
 
+    if (this.config.shadowDomConfig && this.config.shadowDomConfig.enabled && ShadowDomManager.isShadowDomSupported()) {
```

Comment:
Enable the shorthand option as discussed in some comment

```suggestion
    if ((this.config.shadowDom === true || this.config.shadowDom.enabled) && ShadowDomManager.isShadowDomSupported()) {
```

Reply:
[90750c1](https://github.com/bitmovin/bitmovin-player-ui/pull/808/commits/90750c15b99a52cbf477c6e5f37ba21fa9834804)